project/libs/account/blog-user/src/blog-user-module/blog-user.entity.ts

```diff
@@ -0,0 +1,49 @@
+import { compare, genSalt, hash } from 'bcrypt';
+import { Entity } from '@project/shared/core';
+import { StorableEntity, AuthUser} from '@project/shared/core';
+import { SALT_ROUNDS } from './blog-user.constant';
+
+export class BlogUserEntity extends Entity implements StorableEntity<AuthUser> {
```

Comment:
Вот почему не  создается entityFactory

```suggestion
@Injectable()
export class BlogUserEntity extends Entity implements StorableEntity<AuthUser> {
```